test/unit/eval_algorithms/test_faithfulness.py

```diff
@@ -0,0 +1,413 @@
+                verdicts_output='here are the verdicts for the statements:\n1. statement: einstein was born in germany.\nexplanation: the context states that einstein was "a german-born theoretical physicist". this supports that he was born in germany.\nverdict: yes\n2. statement: einstein was born on 20th march 1879.  \nexplanation: the context states that einstein was "born 14 march 1879". this contradicts the statement that he was born on 20th march 1879.\nverdict: no\nfinal verdicts in order:\nyes. no.',
+                expected_score=[EvalScore(name=EvalAlgorithm.FAITHFULNESS.value, value=1 / 2)],
+            ),
+        ],
+    )
+    def test_faithfulness_evaluate_sample(self, test_case):
+        """
+        GIVEN valid inputs
+        WHEN Faithfulness.evaluate_sample is called
+        THEN correct EvalScore is returned
+        """
+        # GIVEN
+        mock_model_runner = Mock()
+        mock_model_runner.predict.side_effect = [
+            (test_case.statements_output, None),
+            (test_case.verdicts_output, None),
+        ]
+        # WHEN
+        eval_algorithm = Faithfulness()
+        actual_score = eval_algorithm.evaluate_sample(
+            model_input=test_case.model_input,
+            model_output=test_case.model_output,
+            target_context=test_case.target_context,
+            judge_model=mock_model_runner,
+        )
+        # THEN
+        assert test_case.expected_score == actual_score
+
+    @pytest.mark.parametrize(
+        "test_case",
+        [
+            TestCaseFaithfulnessEvaluateSample(
+                model_input="Where and when was Einstein born?",
+                model_output="Einstein was born in Germany on 20th March 1879.",
+                target_context="Albert Einstein (born 14 March 1879) was a German-born theoretical physicist, widely held to be one of the greatest and most influential scientists of all time",
+                statements_output="Here are the statements created from the given answer:\nStatement: Einstein was born in Germany.\nStatement: Einstein was born on 20th March 1879.",
+                verdicts_output='here are the verdicts for the statements:\n1. statement: einstein was born in germany.\nexplanation: the context states that einstein was "a german-born theoretical physicist". this supports that he was born in germany.\nverdict: yes\n2. statement: einstein was born on 20th march 1879.  \nexplanation: the context states that einstein was "born 14 march 1879". this contradicts the statement that he was born on 20th march 1879.\nverdict: no\nfinal verdicts in order:\nyes. no.',
+                expected_score=[EvalScore(name=EvalAlgorithm.FAITHFULNESS.value, value=1 / 2)],
+            ),
+        ],
+    )
+    @patch("fmeval.eval_algorithms.faithfulness.get_default_judge_model")
+    def test_faithfulness_evaluate_sample_with_default_judge_model(self, mock_get_default_judge_model, test_case):
```

Comment:
I think you can simplify this test case to simply validate that `get_default_judge_model` is called; there's no need to re-test the same behavior as the test case above.